Deploy/stacks/db/docker-compose.deploy.yml

```diff
@@ -16,16 +16,18 @@ services:
 
   # Blazegraph
   blazegraph:
-    image: docker.cmclinnovations.com/blazegraph:1.0.0-SNAPSHOT
+    image: docker.cmclinnovations.com/blazegraph:1.1.0-510-SNAPSHOT
     container_name: "blazegraph${CONTAINER_NAME_SUFFIX}"
     environment:
-      BLAZEGRAPH_PASSWORD_FILE: /run/secrets/blazegraph_password
+    # Set these environment variables for BASIC authentication
+#      BLAZEGRAPH_PASSWORD_FILE: /run/secrets/blazegraph_password
+#      BLAZEGRAPH_USER: bg_user
       LOG4J_FORMAT_MSG_NO_LOOKUPS: "true"
     pull_policy: if_not_present
     restart: unless-stopped
     # Add a secret to set the password for BASIC authentication
-    secrets:
-      - blazegraph_password
+#    secrets:
+#      - blazegraph_password
```

Comment:
Maybe add a comment to indicate uncomment these two lines if one wants to set authentication.

Reply:
I've added a basic README file.